cvat/apps/engine/serializers.py

```diff
@@ -173,12 +173,38 @@ class Meta:
         fields = ('chunk_size', 'size', 'image_quality', 'start_frame', 'stop_frame', 'frame_filter',
             'compressed_chunk_type', 'original_chunk_type', 'client_files', 'server_files', 'remote_files', 'use_zip_chunks')
 
+    # pylint: disable=no-self-use
     def validate_frame_filter(self, value):
         match = re.search("step\s*=\s*([1-9]\d*)", value)
         if not match:
             raise serializers.ValidationError("Invalid frame filter expression")
         return value
 
+    # pylint: disable=no-self-use
+    def validate_chunk_size(self, value):
+        if not value > 0:
+            raise serializers.ValidationError('Chunk size must be a positive integer')
+        return value
+
+    # pylint: disable=no-self-use
+    def validate_start_frame(self, value):
+        if not value > 0:
+            raise serializers.ValidationError('Start frame must be a non negative integer')
+        return value
+
+    # pylint: disable=no-self-use
+    def validate_stop_frame(self, value):
+        if not value > 0:
```

Comment:
What is the value for stop frame by default?

Reply:
The same as above, I've deleted this validation.